tests/integration/conftest.py

```diff
@@ -933,6 +933,31 @@ def default_aap_credential(
     )
 
 
+@pytest.fixture
+def new_aap_credential(
```

Comment:
Consider a better name for this fixture. Even `another_aap_credential` might be better than `new_aap_credential`.

Reply:
According to our name pattern for test fixtures, "new" prefix means it is a new randomized object so every test use a different instance of the object. Here is a static object of a specific type. 

Looking at how is used this fixture I think it would be better if you just create a batch of credentials for your test instead of combine `new_aap_credential` with `new_aap_credential` 

The fixture can be placed in a new conftest.py file only for tests/integration/analytics unless you expect to be reused outside that scope.

Reply:
New conftest.py file is created